### PR TITLE
add vs ssl cmdline

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: test/e2e/happypath_test.go [hunk 1]
```diff
@@ -292,7 +292,7 @@ var _ = Describe("Happypath", func() {
 				err := envoyInstance.RunWithRole(role, testClients.GlooPort)
 				Expect(err).NotTo(HaveOccurred())
 
-				Eventually(getStatus, "10s", "0.5s").Should(Equal(core.Status_Accepted))
+				Eventually(getStatus, "20s", "0.5s").Should(Equal(core.Status_Accepted))
 			})
```
Comment (rickducott) [inline test/e2e/happypath_test.go]: why did this change?

Comment (yuval-k) [reply test/e2e/happypath_test.go]: i noticed this this test flakes in CI, so i hope this will help to resolve it.

Comment (rickducott) [reply test/e2e/happypath_test.go]: Ah yes here's a ticket tracking FWIW #264

**Pull request merged**